src/poliastro/plotting/_base.py

```diff
@@ -159,6 +186,28 @@ def _plot(self, orbit, *, label=None, color=None, trail=False):
             coordinates, orbit.r, label=label, colors=colors, dashed=True
         )
 
+    def _anim(self, orbit, *, label=None, color=None, trail=False):
```

Comment:
I wonder if we should create something like `AnimatedOrbitPlotter`  which inherits from the `StaticOrbitPlotter` class, as they share most of the methods. In fact, an static plotter should not be able to display animations... Hence, let me ping @astrojuanlu for a further discussion on the topic.

Reply:
it is not about static and all it is only about how long we can get propagated coordinates from that class. because it is the only thing needed to show the animated plots

Reply:
> 
> 
> I wonder if we should create something like `AnimatedOrbitPlotter` which inherits from the `StaticOrbitPlotter` class, as they share most of the methods. In fact, an static plotter should not be able to display animations... Hence, let me ping @astrojuanlu for a further discussion on the topic.

+1

Reply:
Be wary of inheritance: https://en.wikipedia.org/wiki/Composition_over_inheritance